code/lib/cli/src/automigrate/fixes/mdx-1-to-2.ts

```diff
@@ -47,11 +60,23 @@ export const mdx1to2: Fix<Mdx1to2Options> = {
     `;
   },
 
-  async run({ result: { storiesMdxFiles }, dryRun }) {
-    await Promise.all(
-      storiesMdxFiles.map(async (fname) => {
+  async run({ result: { storiesMdxFiles, nonStoriesMdx1Files }, dryRun }) {
+    await Promise.all([
+      ...storiesMdxFiles.map(async (fname) => {
+        const contents = await fse.readFile(fname, 'utf-8');
+        const updated = fixMdxComments(fixMdxStyleTags(contents));
```

Comment:
We should:
 1. we should find all `code-blocks` (which could be in triple backticks, and `~~~`.
 2. we should split the contents-string, on the code-blocks instances
 3. we transform each non-code-block using the string replacer functions we defined above.
 4. join it all back together again

That way we prevent replacing stuff in code-blocks!

@slorber 

I'll try and make this change today.

Reply:
Just pushed a change (just a tad later than I anticipated)